blog/_posts/2021-06-07-cling-declshadow.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Support for redefinitions in Cling, the ROOT's C++ interpreter"
+layout: archive
+author: Javier Lopez Gomez
+---
+
+One of the changes introduced in ROOT 6.20 was to allow users to provide a new definition of a function/variable/class that was already defined. Although ROOT already had two major releases since, we were missing a blog post to make this feature publicly known.
```

Comment:
to me this reads a bit too apologetic, e.g. more positive: "Back in ROOT 6.20, we introduced a big quality-of-life improvement for interactive C++ usage: .... Since then, we made fine-tuned the feature and the feedback we gathered convinced us that it's time for the world to know about cling's shadowing feature!"